src/app/pages/jobs/jobs.component.ts

```diff
@@ -8,12 +8,18 @@ import { Router } from '@angular/router';
 })
 
 export class JobsComponent implements OnInit {
-
+    posts: any;
```

Comment:
Per @taesup, this should not pass review.  Rather than type `any` you should specify explicit type definition.